backend/src/utils/githubUtil.ts

```diff
@@ -271,33 +268,31 @@ const validateOpen = async (
       throw new Error(`User ${username} did not open the pull request ${open.url}.`);
     }
 
-    if (open.createdAt < start || open.createdAt > end) {
+    if (!isWithinDates(open.createdAt, start, end)) {
       const startDate = new Date(start).toDateString();
       const endDate = new Date(end).toDateString();
       throw new Error(`Pull request ${open.url} was not made between ${startDate} and ${endDate}.`);
     }
   });
 
 /** ="at least one of `results` is valid" */
-const atLeastOneValid = (results: ValidationResult[]) =>
+export const atLeastOneValid = (results: ValidationResult[]): boolean =>
```

Comment:
I believe this doesn't get called anywhere anymore (?)

Reply:
Yea, I was thinking they could call that on the `openResults` or `reviewedResults` array of results to get the "overall result" easily, but since those fields dont exist anymore, this won't be directly useful.